pkg/resource/replication_group/custom_update_api.go

```diff
@@ -691,3 +721,40 @@ func (rm *resourceManager) newModifyReplicationGroupRequestPayload(
 
 	return input
 }
+
+// cacheNodeTypeRequiresUpdate retrieves the last requested cacheNodeType saved in annotations and compares them
```

Comment:
if i understand well there will always be only on `cacheNodeType` saved in the `AnnotationLastRequestedCNT` and not multiple ones?

Reply:
This is correct